app/views/includes/modal-free.jade

```diff
@@ -5,23 +5,26 @@ a.modal-action.modal-close.waves-effect.waves-green.btn-flat(title='#{modal.head
     h3 !{modal.head.register_modal}
     .row
         form(name="register", action="/register", method="post").col.s12.center
-            input(type="hidden", name="customerType", value="UPPLICATION")
+            input(type="hidden", name="customer-type", value="UPPLICATION")
```

Comment:
Form names cant be camelcase because they are processed in the backend and we dont want to change the backend.
So this names cant be changed:
- appName
- customerType
- email
- password

You can change the id and the classes names but not the name attr.